Doc/Bib/GapCite.notyet

```diff
@@ -293,3 +293,10 @@ pages = "213--232",
 journal = "Portugaliae Mathematica",
 number = "3",
 }
+
+@article{JGroups2021,
```

Comment:
Thanks - I will pick it up when doing the next update. A minor nitpick: why only initials for one of the authors, and no mathematical notation (`$`) in the title?

Reply:
Because that is the name the author put on the paper himself, it wasn't my decision. I thought inserting `$` might result in parsing errors - I will update this PR and insert them.